Automattic/wp-calypso
# Help Center: Add Help Center on support sites

## Proposed changes

On [March 2nd](https://github.com/Automattic/wp-calypso/pull/73882), team vertex made the HelpCenter available on support pages, unfortunately that PR [caused issues](p58i-e0x-p2) in production. This PR fixes those issues.

## The fix

A. The Help Center was being wrongly loaded on all pages, we have fixed that by checking if the site is a support site.

B. We found that HelpCenter was relying on `@wordpress/interface` which uses `wp-preferences`, `wp-preferences` is not available or not loaded ( we are still not sure ) outside the editor (Guteberg) context. We have removed @wordpress/interface dependency thus wp-preferences

## How to test

- Pull and run this branch
- Navigate to apps/editing-toolkit and run yarn dev --sync to sync your sandbox
- Make sure to sandbox wordpress.com/support
- Navigate to https://wordpress.com/support
- Check if the Help Center is rendered on the admin bar
- Help Center should work fully and normally
